src/index.html

```diff
@@ -84,7 +84,7 @@
                  data-chart-title="Number of loans originated"
                  data-chart-color="blue"
                  data-chart-description="This is the chart description."
-                 data-chart-source="consumer-credit-trends/num_data_AUT.csv">
+                 data-chart-source="consumer-credit-trends/auto-loan/num_data_AUT.csv">
```

Comment:
I have all the back-end updates staged - @mistergone are there any front-end changes or does that just come from the data-chart-source?